packages/manager/apps/cloud/client/app/cloud/project/billing/vouchers/addCredit/cloud-project-billing-vouchers-add-credit-agora.controller.js

```diff
@@ -19,14 +22,9 @@ angular.module('managerApp').controller('CloudProjectBillingVouchersAddcreditAgo
   $onInit() {
     this.amount = 10;
     this.loading = true;
-    return this.$http.get('/order/catalog/formatted/cloud', {
-      serviceType: 'apiv6',
-      params: {
-        ovhSubsidiary: 'US',
-      },
-    }).then((result) => {
-      this.price = get(
-        head(
+    return this.OvhApiOrderCatalogPublic.v6()
+      .get({ productName: 'cloud', ovhSubsidiary: 'US' }).then((result) => {
```

Comment:
Not yours, but we just want the catalog available for US ?

Reply:
This particular controller gets loaded only for US. For the other regions, `cloud-project-billing-vouchers-add-credit.controller.js` gets loaded

Reply:
Even though this is only accessile for US, maybe we could use the user's subsidiary instead that way we can just change the component's restriction if needed

Reply:
This is done. Thanks :)